### DISCUSSION TITLE
Ability to split the query window like SSMS

### DISCUSSION THREAD

**Discussion by gregclerk** [microsoft/azuredatastudio#23671] [Ideas]
SSMS has an icon in the upper right of the vertical scrollbar to split the query window into two panes that scroll independently of each other.  Can this be added to Azure Data Studio?
Before:
![image](https://github.com/microsoft/azuredatastudio/assets/107952826/02570f83-63a3-4362-b125-2c9491a0f5c7)
After:
![image](https://github.com/microsoft/azuredatastudio/assets/107952826/e2286f5d-d634-4d5a-b79f-3805b6d94e83)

Thanks

**Comment by erinstellato-ms** [comment]
Hi @gregclerk can you please also log this as a feature request?  Thanks!

**Comment by mkorsukov** [comment]
It would be nice to see a **vertical** split between the **Query** and **Results** panels in the same tab. It would be much more easier working with long SQL queries and list of data rows.

CC: @erinstellato-ms, @gregclerk
